lib/app/layouts/conversation_view/widgets/text_field/text_field_suffix.dart

```diff
@@ -83,12 +85,12 @@ class _TextFieldSuffixState extends OptimizedState<TextFieldSuffix> {
                 child: isLinuxArm64 ? const SizedBox(height: 40) :
                   !isChatCreator && !showRecording
                   ? CupertinoIconWrapper(icon: Icon(
-                    iOS ? CupertinoIcons.mic : Icons.mic_none,
-                    color: iOS ? context.theme.colorScheme.outline : context.theme.colorScheme.properOnSurface,
-                    size: 20,
+                    iOS ? WaveformIconData(0xe800) : Icons.mic_none,
+                    color: iOS ? CupertinoDynamicColor.resolve(iOSStopIconColor, context) : context.theme.colorScheme.properOnSurface,
+                    size: iOS ? 17 : 20,
```

Comment:
Revert this back to the original code and just make the icon `CupertinoIcons.waveform`.

Reply:
Alright, but I'm changing the size to 24, the waveform icon is pretty small compared to the mic icon 😄